### PR TITLE
complite-clock-move

### PR DESCRIPTION
**Pull request by DimaFlacko**
- [DEMO LINK](https://DimaFlacko.github.io/layout_stop-watch/)

### CODE REVIEW

**Comment by vallerii** [comment]
Good job 👍 
if you want all animation properties can be written in one line

File: src/styles/blocks/clock.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      transform: translate(-50%, -100%);
+      transform-origin: 0 0;
+
+      @include clock-minutes-size;
+      @include position($clock-position);
+
+      animation-name: move;
+      animation-timing-function: linear;
+      animation-duration: 3600s;
+      animation-iteration-count: infinite;
```
Comment (OleksiiBlezniuk) [inline src/styles/blocks/clock.scss:31]: You can combine it to one rule

File: src/styles/blocks/clock.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+  @keyframes move {
+    from {
+      rotate: 0;
+    }
+
+    to {
+      rotate: 360deg;
+    }
+  }
```
Comment (OleksiiBlezniuk) [inline src/styles/blocks/clock.scss:74]: Better to move animation to separate file

**Review by OleksiiBlezniuk — APPROVED** [review]
Good job! Next time move animation (`@keyframes`) to separate file

**Pull request closed without merging**